henryPuma/assignments/twopointers/w1d2/lc1.js

```diff
@@ -0,0 +1,42 @@
+// Problem Statement #
```

Comment:
Good work on the solution. It would be great to see a more detailed pseudocode overview and/or more detailed comments.